src/System.Management.Automation/engine/lang/scriptblock.cs

```diff
@@ -442,24 +433,25 @@ public Collection<PSObject> Invoke(params object[] args)
 
             if (variablesToDefine != null)
             {
-                //
                 // Extract the special variables "this", "input" and "_"
-                //
-                PSVariable located = variablesToDefine.FirstOrDefault(v => string.Equals(v.Name, "this", StringComparison.OrdinalIgnoreCase));
+                PSVariable located = variablesToDefine.FirstOrDefault(
+                    v => string.Equals(v.Name, "this", StringComparison.OrdinalIgnoreCase));
                 if (located != null)
                 {
                     scriptThis = located.Value;
                     variablesToDefine.Remove(located);
                 }
 
-                located = variablesToDefine.FirstOrDefault(v => string.Equals(v.Name, "_", StringComparison.OrdinalIgnoreCase));
+                located = variablesToDefine.FirstOrDefault(
+                    v => string.Equals(v.Name, "_", StringComparison.OrdinalIgnoreCase));
```

Comment:
We could use Ordinal for the ASCII char
```suggestion
                    v => string.Equals(v.Name, "_", StringComparison.Ordinal));
```